asv_bench/benchmarks/groupby.py

```diff
@@ -21,25 +22,27 @@ def setup(self, *args, **kwargs):
     def time_init(self, ndim):
         getattr(self, f"ds{ndim}d").groupby("b")
 
-    @parameterized(["method", "ndim"], [("sum", "mean"), (1, 2)])
-    def time_agg_small_num_groups(self, method, ndim):
+    @parameterized(["method", "ndim", "npg"], [("sum", "mean"), (1, 2), [True, False]])
+    def time_agg_small_num_groups(self, method, ndim, npg):
         ds = getattr(self, f"ds{ndim}d")
-        getattr(ds.groupby("a"), method)()
+        with xr.set_options(use_numpy_groupies=npg):
+            getattr(ds.groupby("a"), method)()
 
-    @parameterized(["method", "ndim"], [("sum", "mean"), (1, 2)])
-    def time_agg_large_num_groups(self, method, ndim):
+    @parameterized(["method", "ndim", "npg"], [("sum", "mean"), (1, 2), [True, False]])
+    def time_agg_large_num_groups(self, method, ndim, npg):
         ds = getattr(self, f"ds{ndim}d")
-        getattr(ds.groupby("b"), method)()
+        with xr.set_options(use_numpy_groupies=npg):
```

Comment:
@Illviljan we could use this approach with `use_bottleneck` too and eliminate one environment for `asv`. It should be close to 2x reduction in total execution time.

Reply:
Nice find!